<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntroducirTemplates</class>
 <widget class="QDialog" name="IntroducirTemplates">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="agrupador">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>371</width>
     <height>171</height>
    </rect>
   </property>
   <widget class="QWidget" name="todos">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>321</width>
       <height>58</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="seleLinSpace">
          <property name="text">
           <string>LinSpace</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="selecLogSpace">
          <property name="text">
           <string>LogSpace</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>NÂº de puntos:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="todasNPuntos"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="individual">
    <widget class="QStackedWidget" name="agrNuDe">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>361</width>
       <height>131</height>
      </rect>
     </property>
     <widget class="QWidget" name="vacia"/>
     <widget class="QWidget" name="pNume">
      <widget class="QTabWidget" name="tabNume">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>361</width>
         <height>141</height>
        </rect>
       </property>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="pDeno">
      <widget class="QTabWidget" name="tabDeno">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>361</width>
         <height>141</height>
        </rect>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>210</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QRadioButton" name="radioNumerador">
        <property name="text">
         <string>Numerador</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioDenominador">
        <property name="text">
         <string>Denominador</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>168</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Aceptar">
      <property name="text">
       <string>ok</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>269</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="Todas">
      <property name="text">
       <string>Todas las variables</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="unaxuna">
      <property name="text">
       <string>Una por una</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>341</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QRadioButton" name="nicols">
      <property name="text">
       <string>Diagrama de Nichols</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="nyquist">
      <property name="text">
       <string>Diagrama de Nyquist</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>301</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QRadioButton" name="CUDA">
      <property name="text">
       <string>CUDA</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Epsilon:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="epsilon"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
